osu.Game/Input/Bindings/RealmKeyBinding.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Input.Bindings;
+using osu.Game.Database;
+using Realms;
+
+namespace osu.Game.Input.Bindings
+{
+    [MapTo("KeyBinding")]
+    public class RealmKeyBinding : RealmObject, IHasGuidPrimaryKey, IKeyBinding
+    {
+        [PrimaryKey]
```

Comment:
Required here to actually take effect (even though it's also specified on the interface). Something I need to follow up on.

Reply:
Unfortunately attributes on interface members aren't inherited at a C# level :(